### PR TITLE
Mentioning updated `transferOwnership` and `renounceOwnership`

### PR DESCRIPTION
**Pull request by b00ste**
## What does this PR introduce?

This PR updated the docs according to the lates updated in the smart contracts around `transferOwnership` and `renounceownerhsip`.

### CODE REVIEW

**Review by Hugoo — APPROVED** [review]

File: docs/standards/smart-contracts/lsp0-erc725-account.md [hunk 1]
```diff
@@ -81,11 +81,14 @@ Return the address of the pending owner that was initiated by [`transferOwnershi
 ### transferOwnership
 
 ```solidity
-function transferOwnership(address newOwner) public {
+function transferOwnership(address newOwner) public
 ```
 
 Initiate an ownership transfer by setting the `newOwner` as `pendingOwner`.
 
+Requirments:
```
Comment (CJ42) [inline docs/standards/smart-contracts/lsp0-erc725-account.md:89]: ~Requirments~ -> Requirements

Also we need to add in the requirements that it **can only be called by the current owner**

File: docs/standards/smart-contracts/lsp0-erc725-account.md [hunk 2]
```diff
@@ -81,11 +81,14 @@ Return the address of the pending owner that was initiated by [`transferOwnershi
...
+- `pendingOwner` cannot be `address(this)`.
```
Comment (CJ42) [inline docs/standards/smart-contracts/lsp0-erc725-account.md:90]: _Suggested changes_

The `newOwner` to be set as the `pendingOwner` cannot be `address(this)`

File: docs/standards/smart-contracts/lsp0-erc725-account.md [hunk 3]
```diff
@@ -96,13 +99,29 @@ Initiate an ownership transfer by setting the `newOwner` as `pendingOwner`.
...
 
 _Triggers the **[OwnershipTransferred](#ownershiptransferred)** event once the new owner has claimed ownership._
 
+### renounceOwnership
+
+```solidity
+function renounceOwnership() public
+```
+
+Renounces ownership of the contract in 2 steps. Can only be called by the owner.
```
Comment (CJ42) [inline docs/standards/smart-contracts/lsp0-erc725-account.md:115]: This is a v**ery sensitive function**. It should be emphasized much more in the docs what is really happening under the hood. Especially the fact that once `renounceOwnership(...)` is called:

- the contract is left without an owner (= the owner is `address(0)`).
- the functions marked as `onlyOwner` cannot be called anymore.

As a good example, the contract API docs of `renounceOwnership` from OpenZeppelin is very cleat about this: https://docs.openzeppelin.com/contracts/4.x/api/access#Ownable-renounceOwnership--

File: docs/standards/smart-contracts/lsp0-erc725-account.md [hunk 4]
```diff
@@ -96,13 +99,29 @@ Initiate an ownership transfer by setting the `newOwner` as `pendingOwner`.
...
+
+First phase of this process will save the current `block.number` in `_lastBlock`.
+
+_Triggers the **[RenounceOwnershipInitiated](#renounceownershipinitiated)** event once the first phase of `renounceOwnership()` is complete._
+
+Second phase of this process will renounce ownership of the contract only if the current `block. number` is bigger than `_lastBlock + 100` and smaller than `_lastBlock + 200` 
+
+_Triggers the **[OwnershipTransferred](#ownershiptransferred)** event once the econd step of `renouneOwnership()` is complete._
```
Comment (CJ42) [inline docs/standards/smart-contracts/lsp0-erc725-account.md:123]: Maybe here we could try to replace `_lastBlock + 100` and `_lastBlock + 200` by some plain words, that explain more what this +100 and +200 mean?

**Review by CJ42 — COMMENTED** [review]
added some reviews comments

File: docs/standards/smart-contracts/lsp0-erc725-account.md (hunk 4 above)
Comment (YamenMerhi) [reply docs/standards/smart-contracts/lsp0-erc725-account.md:123]: agree lets not mention `lastBlock ` as this is an implementation detail

**Pull request merged**